### DISCUSSION TITLE
Theme support

### DISCUSSION THREAD

**Discussion by PiixelNerd** [rafaelvcaetano/melonDS-android#1005] [Ideas]
The emulator could support custom themes, like Drastic and as follows:

My theme:
![Screenshot_20221118-083628_DraStic](https://user-images.githubusercontent.com/93597875/202696680-966be448-ff25-491c-952a-0730c30b4855.png)

Internet's theme:
![images](https://user-images.githubusercontent.com/93597875/202696710-cf3044ed-3408-4144-80e8-c1849cc81910.jpeg)

**Comment by rafaelvcaetano** [comment]
This is already possible. You'll need to create a custom layout and then add backgrounds. Here is an example of a custom layout with a background:
![image](https://user-images.githubusercontent.com/9086379/202732593-38022071-9040-445f-b244-152a4ccd9cbb.png)

**Reply by PiixelNerd** [reply]
thank you very much, I didn't know.  I imagined that the background function was just to simulate a background like wallpaper.  I didn't think about putting an image with buttons as a background, thanks!
